app/assets/stylesheets/components/_header.scss

```diff
@@ -0,0 +1,52 @@
+.header {
+  @include header-gradient;
+  position: relative;
+  top: -30px;
+
+  padding: 15px 0;
+  background-color: $white-100;
+  box-shadow: 0 0 0 1px rgba(0, 0, 0, 0.01);
+
+  .container {
+    display: flex;
+    align-items: center;
+    justify-content: space-between;
+  }
+
+  &__left {
+    display: flex;
+    align-items: center;
+
+    h3,
+    span {
+      font-size: 1rem;
+      font-weight: 300;
+    }
+
+    h3 {
+      margin-bottom: 0;
+    }
+
+    span {
+      margin-left: 5px;
+      opacity: 0.8;
+      color: $primary;
+    }
+  }
+
+  &__right {
+    width: 100%;
+    max-width: 100px;
+
+    form {
+      display: flex;
+      justify-content: flex-end;
+      width: 100%;
+      max-width: 100px;
+
+      button {
+        padding: 5px 0;
+      }
+    }
```

Comment:
Should we make it part `form` component?

Reply:
I've removed this since the header is basically empty and serving as a light bar for the application 😄 Updated in https://github.com/aetheryna/google-scraper-ruby/pull/35/commits/f347b499d2460bbd6f358239317f5953a1edd419